Client/Frontend/Widgets/SnackBar.swift

```diff
@@ -0,0 +1,225 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+import Foundation
+import Snap
+
+// Small wrapper class for buttons in a snackbar. Takes a title and a callback.
+// Inherits from NSObject so that this can work as a selector target
+class SnackButton : NSObject {
+    let title: String
+    let callback: (bar: SnackBar) -> Void
+    private var bar: SnackBar!
+
+    init(title: String, callback: (bar: SnackBar) -> Void) {
+        self.title = title
+        self.callback = callback
+    }
+
+    func onClick() {
+        println("onClick")
+        callback(bar: bar)
+    }
+
+}
+
+/* Presents some information to the user. Can optionally include some buttons and
```

Comment:
If you use `///` here then Xcode will pick it up as code docmentation that will show up in the Quick Help Inspector.